core/src/main/java/org/elasticsearch/cluster/ClusterName.java

```diff
@@ -33,6 +33,8 @@
     public static final Setting<ClusterName> CLUSTER_NAME_SETTING = new Setting<>("cluster.name", "elasticsearch", (s) -> {
         if (s.isEmpty()) {
             throw new IllegalArgumentException("[cluster.name] must not be empty");
+        } else if(s.contains(":")){
```

Comment:
There should be an extra space between the `if` and the `(`, and the `))` and the `{`.